### PR TITLE
October 25th, 2016 release notes

### PR DESCRIPTION
**Pull request by orangejulius**
With a nice separation between features and "code level" changes.

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -4,6 +4,18 @@
...
 > specifically for the hosted instance of Pelias run by Mapzen, Mapzen Search.
 
+## 24 October 2016
+
+### New features and fixes
+* The `/v1/autocomplete` endpoint now supports [boundary.rect](https://mapzen.com/documentation/search/search/#search-within-a-rectangular-region) just like `/v1/search`
+* Labels for administrative areas should be [improved in a few cases](https://github.com/pelias/whosonfirst/pull/139)
+
+### Code level changes
+* All Pelias code now requires Node.js 4.0 or newer! This means we can start using ES6 features. `let` us have a party to celebrate! :tada:
```
Comment (trescube) [inline release-notes.md]: Oof

Comment (orangejulius) [inline release-notes.md]: if you have a better ES6 pun I'm all ears

File: release-notes.md [hunk 2]
```diff
@@ -4,6 +4,18 @@
...
+* Label generation code has been extracted into [pelias-label](https://github.com/pelias/labels)
```
Comment (trescube) [inline release-notes.md]: should be `pelias-labels`

File: release-notes.md (hunk 1 above)
Comment (orangejulius) [inline release-notes.md]: If you have one, i promise to use it

File: release-notes.md (hunk 2 above)
Comment (orangejulius) [inline release-notes.md]: fixed!

**Pull request merged**